packages/docs/src/styles/_props.scss

```diff
@@ -33,12 +33,15 @@
 }
 
 .docs-prop-tags {
-  margin: $pt-grid-size 0 0;
+  $tag-top-margin: ($pt-grid-size / 10) * 3;
```

Comment:
honestly I'd prefer if you just said `3px` here cuz it's less inscrutable ("scrutable").

Reply:
I s'pose. But what what if one day white space becomes cool again and we bump up `$pt-grid-size` to 💯 ?

Reply:
`$pt-grid-size * 0.3`

Reply:
@llorca Yeah that'd be better. Next week.